.github/workflows/linux-eic-shell.yml

```
@@ -221,6 +221,9 @@ jobs:
 
   check-overlap-tgeo:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        option: ['', 's']
```

Apparently the empty string is failing to get parsed correctly... but any letter will do.
```suggestion
        option: ['m', 's']
```